.changeset/config.json

```diff
@@ -12,5 +12,9 @@
     "access": "public",
     "baseBranch": "main",
     "updateInternalDependencies": "patch",
-    "ignore": []
+    "ignore": [],
+    "snapshot": {
+        "useCalculatedVersion": true,
+        "prereleaseTemplate": "{tag}.{datetime}"
```

Comment:
This configures the suffix for the snapshot releases, using a template with placeholders.
I decided for `tag.datetime`, which seems to be a common pattern. But we could have used:
- `{tag}` - the name of the snapshot tag, as specified in `--snapshot something`
- `{commit}` - the Git commit ID
- `{timestamp}` - Unix timestamp of the time of the release
- `{datetime}` - date and time of the release (14 characters, for example, `20211213000730`)

> Note: because we are using `{tag}` as placeholder, we can't call `--snapshot` with an empty tag name - this will result in error.

Reply:
What's nice too is I think the tag.datetime syntax matches how Garth did the snapshots for foundations in the spectrum-tokens repo too! Cross-team synchronicity!

Reply:
@rubencarvalho what do you think about the tag being the PR number or the number is prepended the custom tag?This would make it easier to understand which beta snapshots are associated with PR fixes.

Reply:
@caseyisonit There's no asset for PR number because there's no guarantee every tag will be connected to a PR but could use the commit.